napari/_qt/tests/test_qt_layerlist.py

```diff
@@ -83,10 +83,10 @@ def test_creating_empty_view(qtbot):
     # Check that the layers model has been appended to the layers view
     assert view.layers == layers
 
-    # Check that vbox_layout only contains one QtDivider and one spacer
-    assert view.vbox_layout.count() == 2
-    assert check_layout_layers(view.vbox_layout, layers)
-    assert check_layout_dividers(view.vbox_layout, 0)
+    # Check that vboxLayout only contains one QtDivider and one spacer
```

Comment:
I don't think this variable name should be updated if not necessary. It's tricky working with PyQt in this space, but we have e.g. grid_layout and main_layout, and I think in general that works fine: use PEP8 *except* where Qt forces your hand, and in method names that are meant to interact with Qt directly. For variable names, I think we should stick with pythonic style.

Reply:
ok reverted - i was trying to make things consistent, but agreed then i used `grid_layout` too - lol - we have this open issue #6 that i'd really like to resolve soon, but that can come after this get's merged. we might need to make another pass and make everything consistent